lib/ansible/module_utils/rabbitmq.py

```diff
@@ -193,4 +194,8 @@ def basic_publish(self):
         if args['exchange'] is None:
             args['exchange'] = ''
 
-        return self.conn_channel.basic_publish(**args)
+        try:
+            self.conn_channel.basic_publish(**args)
+            return 1
```

Comment:
Why `1`/`-1` and not something like `True`/`False`?

Reply:
No good reason.  I've updated to `True` / `False`.  Thank you!